src/ophys_etl/schemas/dense_roi.py

```diff
@@ -0,0 +1,51 @@
+from marshmallow import Schema
+from marshmallow.fields import (List, Str, Float, Int, Bool)
+
+
+class DenseROISchema(Schema):
```

Comment:
I like the idea of keeping this in it's own file, I'm just not sure about the name DenseROISchema (it seems relatively ambiguous as I can see people interpreting it multiple ways). Maybe CorrectedDenseROISchema or ProcessedDenseROISchema, the confusion to me comes from having it say only Dense but also including exclusion labels and correction values. In my head DenseROI would indicate exclusively the mask data.

Reply:
@isaak-willett Was the included docstring for that schema not sufficient?

```
This ROI format is the expected output of Segmentation/Binarization
and the expected input of Feature_extraction/Classification.
```

Reply:
I'm not sure that external users will read the inline docstrings, but I could totally be wrong. I'm not sure how I would resolve this necessarily as I feel like the format is dependent on LIMs and not necessarily on that being the expected input format of the workflow steps. I'm probably overthinking this tbh, my chief concern is we do a lot of transformations between two formats, ie: we expect one format but almost immediately transform it into another format to work on it. What I'm concerned about is that the motivation for this is very gargled for any external user, they could look at this and see this format and that we immediately move away from that when processing occurs and get very confused as they have no insight into LIMS.

I think it's beside the point though since this PR does not appeared to be targeted at changing this, I must have gotten confused on which PR went to which.